api/buildings/static/js/admin/here_map.js

```diff
@@ -0,0 +1,167 @@
+(function($) {
+  // We'll insert the map after this element:
+  var prev_el_selector = "#id_lng";
+
+  // The input elements we'll put lat/lon into and use
+  // to set the map's initial lat/lon.
+  var lat_input_selector = "#id_lat";
+  var lon_input_selector = "#id_lng";
+
+  // The input elements we'll put the address string and country code into.
+  var street_input_selector = "#id_address";
+  var street_number_input_selector = "#id_street_number"
+  var locality_input_selector = "#id_locality";
+
+  // We'll insert the map after this element:
+  var prev_el_selector = "#id_lng";
+
+  // Initial zoom level for the map.
+  var initial_zoom = 12;
+
+  // Initial zoom level if input fields have a location.
+  var initial_with_loc_zoom = 18;
+  var initial_lat = 44.4268;
+  var initial_lon = 26.1025;
+
+  function distance(lat1, lon1, lat2, lon2) {
```

Comment:
Could we [use this](https://developer.here.com/documentation/android-premium/3.18/dev_guide/topics/distance-calculations.html) instead ?

Reply:
Since the link was for Android SDK, I looked for the equivalent functionality in JavaScript SDK: https://developer.here.com/blog/distance-calculations-between-points-with-the-here-javascript-sdk I tried implementing the code as explained there, by creating 2 markers and computing the distance between them, but for some reason the code was failing. My assumption is that the article was written for an older version of the SDK and the method `getPositition()` was replaced.

Reply:
Nvm, I found the solution :)